docs/docs/task-handling.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Task Handling
+---
+
+Marathon handles tasks in the following ways. 
+
+Can a user access these messages? How? Via the API? The DC/OS CLI?
+
+# Terminal states
+
+```
+case TASK_ERROR => Error
```

Comment:
I think it tries forever but I'm not sure. @aquamatthias can probably answer this.